### ISSUE TITLE
Heart rate sensor not showing up

### ISSUE THREAD

**Issue by parkerlreed** [dlenski/ttbindec#1]
I notice the 0x23 code seems to be unknown/possibly related to GPS. Is it possible this contains heart rate data? 

I have a run where I had a Polar attached (Runner on firmware 1.8.52) and the only two extra data elements I have are 0x23 and 0x42. 0x42 is always zero so that can't be it. And it doesn't appear in the main GPS data block either.

```
0004a1f9 22 FILE_GPS_RECORD(latitude=xxxxxxxx, longitude=-xxxxxxxx, heading=21114, gps_speed=4, timestamp=1584232723, calories=322, instant_speed=0.0, cum_distance=23302.3027344, cycles=0)
0004839d 23 UNKNOWN_0x23(unhexlify('fd04f6020706111c131d1b1a19000000000402'))
000483b1 42 UNKNOWN_0x42(unhexlify('00'))
```

**Comment by dlenski** [comment]
> I notice the 0x23 code seems to be unknown/possibly related to GPS. Is it possible this contains heart rate data?

I myself don't have a TT device with any kind of internal or external HRM, but I believe heart rate records are represented by the 0x37 tag:

https://github.com/dlenski/ttbindec/blob/master/defs.py#L273-L294

I doubt that these records have anything to do with heart rate. I wrote the following in the README, and I was basing this off of looking at files from a Runner v1 with no HRM:

> * `0x23` Appears subsequent to **every single** `FILE_GPS_RECORD`. Values seem to
change suddenly in areas where GPS signal is shaky, and stabilize
towards the end of long runs.
> 
> ```
> 23 RECORD_LENGTH(tag=35, length=20)    [ UNKNOWN_0x23 ]
> ```

**Comment by dlenski** [comment]
Could you share one of the TTBIN files generated by Runner v1 + Polar HRM?

Does the official TT software extract the expected HR data?

**Comment by parkerlreed** [comment]
I may have looked at the wrong run. I left that laptop that I saved them to at home today so I'll let you know when I get back.

Also the only official software for Linux appears to just sync the saves to the cloud. I've never bothered with it.

Thanks.

**Comment by dlenski** [comment]
Basically, unless you have some evidence that there is heart rate data in these files which…

(a) ttbindec and [ryanbinns/ttwatch](https://github.com/ryanbinns/ttwatch/) fail to extract
(b) but some other software _does manage to extract_ (probably TT's official Windows/Mac/iOS/Android clients)

… then I have no reason to think these 0x23 mystery tags have anything to do with heart rate.

As I speculate in the README, they probably have something to do with the status of the GPS chip.

**Comment by parkerlreed** [comment]
Yeah I get that. I was just speculating since I didnt see it elsewhere.

I don't have a good run to test right now. One wasnt with the sensor and the other I think did but has my home address. I'll have to get another run where I can control the variables (somewhere random and know for sure I have the sensor running)

**Comment by parkerlreed** [comment]
Also if it gets deleted from the watch on sync, I assume I can't test the exact same run in two different software?

**Comment by dlenski** [comment]
The Windows and Android software leave behind copies of the `.ttbin` files after they sync.
